setup.py

```diff
@@ -45,7 +45,7 @@ def has_environment_marker_support():
 def main():
     extras_require = {}
     install_requires = [
-        'py>=1.4.33',
+        'py>=1.5.0',
```

Comment:
please rebase to avoid this bit creating issues

Reply:
Definitely, I'm still working on this.